comparison.md

```diff
@@ -109,19 +106,19 @@ AssertJ provides assertions for several libraries' types. As of this writing,
 Truth includes assertions for [Guava] and [Protocol Buffers].
 
 Both have third-party extensions, such as for Android types
-([AssertJ][AssertJ-Android], [Truth][Truth-Android]). I don't have a feel for
-the overall size of each ecosystem.
+([AssertJ][AssertJ-Android], [Truth][Truth-Android]).
+It's unclear how large each ecosystem is overall.
 
 ### Platform support (Android, GWT)
 
 AssertJ supports Android (though I had to use 2.x because the dexer rejected
-3.x, even when I used only `Java6Assertions`). Possibly this was an issue with
-my build setup.
+3.x, even when using only `Java6Assertions`). Possibly this was an issue with
+the build setup.
 
 Truth supports Android in all its versions. The downside is that it requires you
 to look in a separate class for Java 8 assertions.
 
-Truth also supports [GWT], but we haven't made the necessary artifacts available
+Truth also supports [GWT], but the necessary artifacts haven't been made available
```

Comment:
I think this is no longer true? @cpovirk?

Reply:
Oops, you're right. This was not true even when I wrote this! I forgot that we generate `truth-gwt` artifact without using a separate project like we do for Guava.